doc/source/how_to_install.md

```diff
@@ -28,9 +28,18 @@ Updating packages with `pip` (and sometimes `mamba`) can break your installation
 
 ## Installing for contributors
 
+### With ``mamba``
+
 ```bash
 git clone https://github.com/GlacioHack/xdem.git
 mamba env create -f xdem/dev-environment.yml
 ```
 
+### With ``pip``
+
```

Comment:
I would add documentation for this PR saying that GDAL is needed for development. Maybe in a section with make test ?